projects/core/src/lib/datasets/add-data/add-data.component.ts

```diff
@@ -56,7 +55,7 @@ export class AddDataComponent implements OnInit {
         return {
             name: 'Layers',
             description: 'Collection of existing layers',
-            iconSrc: createIconDataUrl('Layer Collection'),
+            icon: 'sort',
```

Comment:
wasn't this supposed to be `layers`?

Reply:
Oops, fixed that in 0a65ceefb42d3a9b9bc603e0ea676a8a9eb358fb